src/keboola_mcp_server/links.py

```diff
@@ -27,6 +27,26 @@ async def from_client(cls, client: KeboolaClient) -> 'ProjectLinksManager':
         project_id = await client.storage_client.project_id()
         return ProjectLinksManager(base_url, project_id)
 
+    def _detail_link(self, label: str, name: str, url: str) -> Link:
+        """Builds object detail link."""
+        return Link(type='ui-detail', title=f'{label}: {name}', url=url)
+
+    def _dashboard_link(self, title: str, url: str) -> Link:
+        """Builds object dashboard link."""
+        return Link(type='ui-dashboard', title=title, url=url)
+
+    def _docs_link(self, title: str, url: str) -> Link:
+        """Builds object documentation link."""
+        return Link(type='docs', title=title, url=url)
```

Comment:
these could be factory methods directly under `Link` class (`Link.detail_link(...)`)